thunder/images/images.py

```diff
@@ -1,5 +1,7 @@
 import logging
 from numpy import ndarray, arange, amax, amin, size, asarray, random, prod
+# Name changes between Python 2 and 3
+from six.moves import xrange
```

Comment:
cool, great, totally see that reasoning, but also totally agree there are few places where that performance difference will matter.